app/products/calls/state/current_call.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {useEffect, useState} from 'react';
+import {BehaviorSubject} from 'rxjs';
+
+import {CurrentCall} from '@app/products/calls/types/calls';
+
+const currentCallSubject = new BehaviorSubject<CurrentCall | null>(null);
+
+// Used internally, only exported for tests (not exported from the module index).
+export const setCurrentCall = (currentCall: CurrentCall | null) => {
```

Comment:
As I said on a previous comment, can be exported in a "for test only" export.

Reply:
Done, thanks.